addons/l10n_ar/models/res_partner.py

```diff
@@ -52,6 +52,24 @@ def _compute_l10n_ar_vat(self):
         remaining = self - recs_ar_vat
         remaining.l10n_ar_vat = False
 
+    @api.depends('l10n_latam_identification_type_id', 'l10n_ar_vat')
+    def _compute_is_company(self):
+        "True if partner is considered a company in Argentina, based on Identification Type and CUIT prefix."
+        l10n_ar_partners = self.filtered(lambda p: p.vat and (
```

Comment:
Is "VAT" required really here ? Even if 'na', 'NA', ... ? As the computation does not use vat but l10n_ar_vat and afip code. Do we expect people to also set a VAT ?